### DISCUSSION TITLE
마크다운, 어디까지 써보셨나요?

### DISCUSSION THREAD

**Discussion by chongin12** [DeveloperAcademy-POSTECH/.github#22] [Idea]
여러분들은 마크다운 어디까지 써보셨나요? Github, Notion, Discord, Slack 등 여러 툴을 사용해보신 분들이라면 마크다운의 필요성을 느끼셨을 것이라고 생각됩니다!

많은 사이트에서 마크다운에 대해 소개가 많이 되어있지만, 그래도 한번에 모아보면 좋을 것 같아서 공유합니다!

(이전에 사용했던 발표자료라서 슬라이드마다 자세한 설명은 되어있지 않습니다..!)

![Markdown 어디까지 써보셨나요 002](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/766a9549-a3e8-488d-b5f5-b85b3ff49402)
![Markdown 어디까지 써보셨나요 003](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/c6f07c75-b2ba-4eb7-9f42-758faf08d3f9)
![Markdown 어디까지 써보셨나요 004](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/756738d0-8efd-421a-afb5-f72a0832dd74)
![Markdown 어디까지 써보셨나요 005](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/b9260281-dffa-4124-9f1c-1d13763adefd)
![Markdown 어디까지 써보셨나요 006](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/52bb3740-f082-4fd2-b36e-7f2a8d489133)
![Markdown 어디까지 써보셨나요 007](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/09128e49-1bfa-4803-8a74-552bfe95deb1)
![Markdown 어디까지 써보셨나요 008](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/53db44da-1a1f-4214-8f9d-53ce08aef0b2)
![Markdown 어디까지 써보셨나요 009](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/76178212-3082-468f-89f3-25211eb5d6d7)
![Markdown 어디까지 써보셨나요 010](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/5266d6cb-9445-4c57-9e69-5aa7dd204adb)
![Markdown 어디까지 써보셨나요 011](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/bf580e65-7132-43dd-9fc4-72384c898f19)
![Markdown 어디까지 써보셨나요 012](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/357a718d-129a-4d79-b968-69058dba7bf9)
![Markdown 어디까지 써보셨나요 013](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/7f6b5e55-2862-40c2-8412-15b75c62edc9)
![Markdown 어디까지 써보셨나요 014](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/4cc0244b-028e-4da4-8918-09f1364c8a78)
![Markdown 어디까지 써보셨나요 015](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/7d626857-8a1c-43c5-817f-fcf969fd7257)
![Markdown 어디까지 써보셨나요 016](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/956ddf2e-e81f-4c7b-bb0c-fb2c47356a2a)
![Markdown 어디까지 써보셨나요 017](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/025574a4-0e95-4555-afce-ead992250328)
![Markdown 어디까지 써보셨나요 018](https://github.com/DeveloperAcademy-POSTECH/.github/assets/19565940/b8c50331-0196-4283-83e3-4aa6c9cd7125)

**Comment by seoyounghan** [comment]
와 리드미를 작성하다가 어려움을 겪은 저에게는 매우 필요한 글이였습니다! 감사합니다 모수

**Comment by anjiniii** [comment]
안녕하세요, 3기 이오입니다!

오 리스트를 사용할 때 번호를 순서대로 쓰지 않아도 순서대로 나오는군요!!
신기해서 한번 해봤습니다 ㅎㅎ

1. 1번 항목
4. 4번 항목
8. 8번 항목
<img width="170" alt="image" src="https://github.com/DeveloperAcademy-POSTECH/.github/assets/117328806/d4def8ed-de44-413d-bfdc-adfce5824a04">

**Comment by JongminAhn** [comment]
다음엔 제가 리드미 작성을 해봐야겠다는 생각이 들었어요!

**Comment by lingostar** [comment]
전 MacDown 이라는 앱을 사용한답니다. 바로 옆에서 프리뷰를 볼 수 있어서 편해요

https://macdown.uranusjr.com

**Comment by Jaehwa-Noh** [comment]
Onlline에서도 볼 수 있습니다.
https://markdownlivepreview.com/
